### DISCUSSION TITLE
There is an error in the cobalt.

### DISCUSSION THREAD

**Discussion by fengchun1234** [Raytwo/Cobalt#44] [Issues]
Error Code: 2-0069
Cobalt v1.1.1 has panicked at 'crates \cobalt\src\catalog.rs:22:58' with the following message
Could not read sd:/engage/cache.lut', make sure to download it on Github: Custom { kind: Other, error: "Result code: 514"}

Could you please see if it can be resolved? Thank you!

**Comment by Raytwo** [comment]
Hello, way too late on replying but you just did not take the file in release.zip
